mne/io/brainvision/tests/data/test_highpass_hz.vhdr

```diff
@@ -0,0 +1,103 @@
+Brain Vision Data Exchange Header File Version 1.0
+; Data created by the Vision Recorder
```

Comment:
Instead of adding a bunch of files can you just do some read/string editing/write (to temporary directory)? It should cut down on size and maybe be more readable anyway since it's clearer what differs. WDYT?